locations/spiders/molton_brown.py

```
@@ -0,0 +1,40 @@
+                for times in feature["store_business_hours"]:
+                    try:
+                        if times["open_24hrs"] is not False:
+                            oh.add_range(DAYS_FULL[int(times["week_day"]) - 1], "00:00", "23:59")
+                        else:
+                            oh.add_range(
+                                DAYS_FULL[int(times["week_day"]) - 1],
+                                times["open_time"],
+                                times["close_time"],
+                                time_format="%I:%M %p",
+                            )
+                    except:
```

Ruff (0.16.1)

[error] Do not use bare `except`

(E722)